plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -73,9 +73,11 @@ export class EventsProcessor {
             if (!team) {
                 throw new Error(`No team found with ID ${teamId}. Can't ingest event.`)
             }
-
             if (data['event'] === '$snapshot') {
                 if (team.session_recording_opt_in) {
+                    if (await this.teamManager.isAboveUsageLimit(team.organization_id, 'recordings')) {
```

Comment:
Missing _unit_ test in plugin-server/tests/worker/ingestion/process-event.test.ts